src/main/steps/applicant/yourdocuments/alldocuments/alldocuments/tasklistalldocuments.ts

```diff
@@ -633,17 +638,20 @@ const getApplicantAohAndViolence = (applicant: Applicant, taskListItems, userCas
       '<nameapplicantxxxxx>',
       applicant.value.firstName + ' ' + applicant.value.lastName
     ),
-    href: getApplicantAllegationsOfHarmAndViolence(userCase) === true ? URL.ALLEGATION_OF_HARM_VOILENCE : '#',
```

Comment:
not necessary to check this condition explicitly

Reply:
Done